paddle/gserver/evaluators/ChunkEvaluator.cpp

```diff
@@ -243,6 +242,53 @@ class ChunkEvaluator : public Evaluator {
     if (tag == tagSingle_) return true;
     return false;
   }
+
+public:
+  // three metrics: precision, recall and F1-score
+  void getNames(std::vector<std::string>* names) {
```

Comment:
Shall we push `precision`, `recall`, `F1-score` directly into `names?
 It is simpler and more computationally efficient.